src/app/app-routing.module.ts

```diff
@@ -11,6 +12,7 @@ const routes: Routes = [
   { path: '', redirectTo: '/homepage', pathMatch: 'full' },
   { path: 'homepage', component: HomePageComponent },
   { path: 'orderspage', component: OrdersPageComponent },
+  { path: 'ownerrestaurantspage', component: OwnerRestaurantsComponent },
```

Comment:
Maybe just my-restaurants or just restaurants instead of ownerrestaurantspage. It looks odd in the url